feedback/templates/manage/_feedbacks_as_list.html

```diff
@@ -7,6 +7,14 @@
 		feedbacks created with get_feedback_dict
 {% endcomment %}
 
+{% if request.GET.paginate_by %}
+	<script>
```

Comment:
I was thinking that this script would not be needed if the selected option were set in the HTML attributes when the select menu is rendered in the template `_pagination.html`. However, at the moment, it looks inconvenient to set the attribute in the html template since it would require if statements inside each `<option>` to set its attribute only when that option is already selected in `request.GET.paginate_by`.